app/services/form_to_resource_service.rb

```diff
@@ -7,6 +7,9 @@ class << self
     #  @param [Work] work params will be applied to. Utilizes the work for old values if needed.
     #
     # @return [PDCMetadata::Resource] Fully formed resource containing updates from the user
+    #
```

Comment:
@jrgriffiniii there is no change to the method, why are these new disables required?

Reply:
I am very sorry, these must have been tracked from an earlier commit. I have removed these.